vent/core/rq_worker/file_watch.py

```diff
@@ -54,13 +55,24 @@ def file_queue(path, template_path="/vent/"):
             if config.has_option(section, 'settings'):
                 try:
                     options_dict = json.loads(config.get(section, 'settings'))
-                    for option in options_dict:
-                        if option == 'ext_types':
-                            ext_types = options_dict[option].split(',')
-                            for ext_type in ext_types:
-                                if path.endswith(ext_type):
-                                    images.append(image_name)
-                                    configs[image_name] = {}
+                    process_file = False
+                    # check if this tool should look at files in base dir
+                    if 'process_base' in options_dict:
```

Comment:
the default should be process the base only even if the `process_base` option isn't there.